deploy/olm-catalog/kubevirt-ssp-operator/0.16.0/kubevirt-ssp-operator.v0.16.0.clusterserviceversion.yaml

```diff
@@ -1,8 +1,10 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  creationTimestamp: null
-  name: kubevirt-ssp-operator.v0.1.1
+  annotations:
+    alm-examples: '[{"apiVersion":"kubevirt.io/v1","kind":"KubevirtCommonTemplatesBundle","metadata":{"name":"kubevirt-common-template-bundle"},"spec":{"version":"v0.4.1"}},{"apiVersion":"kubevirt.io/v1","kind":"KubevirtTemplateValidator","metadata":{"name":"kubevirt-template-validator"},"spec":{"version":"v0.1.1"}}]'
+    capabilities: Basic Install
+  name: kubevirt-ssp-operator.v0.16.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
```

Comment:
I don't believe you have api services, but rather [owned custom resource definitions](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#owned-crds).

Reply:
The brackets should have been a clue.

Reply:
OK, I'll keep the apiservicedefinition as-is